queries.sql

```diff
@@ -154,4 +154,47 @@ GROUP BY species;
 SELECT species, AVG(escape_attempts) 
 FROM animals
 WHERE date_of_birth BETWEEN 'January 1, 1990' AND 'December 31, 2000'
-GROUP BY species;
\ No newline at end of file
+GROUP BY species;
+
+/*What animals belong to Melody Pond?*/
+SELECT animals.name, owners.full_name
+FROM animals
+JOIN owners ON owners.id = animals.owner_id
+WHERE owners.full_name LIKE 'Melody Pond';
+
+/*Animals that are pokemon*/
+SELECT animals.name, species.name AS species_name
+FROM animals
+JOIN species ON species.id = animals.species_id
+WHERE species.name LIKE 'Pokemon';
+
+/*Owners and their aniamls*/
+SELECT animals.name, owners.full_name AS owner_name
+FROM owners
+LEFT JOIN animals ON owners.id = animals.owner_id
+
+/*Number of animals per specie*/
+SELECT species.name, COUNT(animals.name) AS animal_per_specie 
+FROM animals
+JOIN species ON species.id = animals.species_id
+GROUP BY species.name
+
+/*List all Digimon owned by Jennifer Orwell*/
+SELECT owners.full_name, animals.name
+FROM animals
+JOIN owners ON animals.species_id = owners.id
+WHERE  animals.name LIKE '%mon';
```

Comment:
- The query says `List all Digimon owned by Jennifer Orwell`. Kindly check and modify the query above as the only `Digimon` being owned by Jennifer Orwell is `Gabumon`.

This 👇 is what I got when I ran our query.

![](https://i.imgur.com/4Sjsc5G.png)

- For ease, 😉 you can write the query like so 👇 
```
 SELECT animals.* FROM animals
   LEFT JOIN owners
   ON animals.owner_id = owners.id
   LEFT JOIN species
   ON animals.species_id = species.id
   WHERE owners.full_name = 'Jennifer Orwell'
   AND species.name = 'Digimon';
```